source/_includes/header.liquid

```diff
@@ -0,0 +1,35 @@
+<header
+  class="sl-r-banner"
+  itemtype="http://schema.org/WPHeader"
+  itemscope="itemscope"
+  role="banner">
+  <div class="sl-c-alert sl-c-alert--info" style="background: #000; padding: 3rem 0; text-align: left;">
+    <div class="sl-l-container">
+      <h2 style="margin-top: 0;">Black Lives Matter</h2>
```

Comment:
@stacyk Throughout this PR I just copied over markup directly, but feel free to clean it up however you like. I'm not sure why they used inline styles here? 🤷

Reply:
I'm cleaning this up in this PR now @jgerigmeyer